.github/workflows/create-next-pr.yml

```diff
@@ -0,0 +1,34 @@
+name: "Create Next PR"
+
+on:
+  workflow_dispatch: {}
+  workflow_call: {}
```

Comment:
Is the CDKTF pre-release workflow going to call this or what is this for?

Reply:
not yet, but I wanted to keep the option open without having to do two PRs